lms/static/scripts/frontend_apps/components/dashboard/SyncGradesButton.tsx

```diff
@@ -70,7 +89,17 @@ export default function SyncGradesButton({
       return (
         <>
           Syncing grades
-          <SpinnerCircleIcon className="ml-1.5" />
+          <div
+            className={classnames(
+              'border-solid border-l border-grey-5',
+              // Compensate the button's padding, so that this grows from top to
+              // bottom
+              '-my-2 self-stretch',
```

Comment:
I considered using a ButtonGroup here, but <) it does not currently support the `Button` component, only the `IconButton` one, and b) it would require more changes here to dynamically render a different component tree.

I decided to be pragmatic and go with this small workaround. If we solve point a) at some point, we can revisit this.